loopdb/loopout.go

```diff
@@ -45,6 +45,10 @@ type LoopOutContract struct {
 	// MaxPrepayRoutingFee is the maximum off-chain fee in msat that may be
 	// paid for the prepayment to the server.
 	MaxPrepayRoutingFee btcutil.Amount
+
+	// MaxSwapWaitTimeUnix is the maximum time in unix seconds the client
+	// allows the server to wait before publishing the on chain swap.
+	MaxSwapWaitTimeSec int64
```

Comment:
Could we name this `SwapDeadline` or at least `MaxSwapWaitTimestamp`? As far as I understood it, it no longer contains the relative number of seconds to wait but an absolute timestamp at this point. So it's a bit confusing that it still has the same name as in the gRPC message.

Reply:
I liked the "deadline" idea, so converted all these to `SwapPublicationDeadline`, and using `time.Time`: https://github.com/lightninglabs/loop/pull/75/commits/1cff25ac6aed4fee758741d6fef8c46b4f017cce